locustfile.py

```diff
@@ -0,0 +1,237 @@
+        self.client.get("/static/khan-exercises/css/khan-exercise.css")
+        self.client.get("/static/js/backbone.js")
+        self.client.get("/static/js/distributed/exercises.js")
+        self.client.get("/static/khan-exercises/khan-exercise.js")
+        self.client.get("/securesync/api/user/status/")
+        self.client.get("/static/khan-exercises/local-only/katex/fonts/fonts.css")
+        self.client.get("/static/khan-exercises/local-only/katex/katex.css")
+        self.client.get("/static/khan-exercises/local-only/jquery-migrate-1.1.1.js")
+
+
+    @task(1)
+    def index(self):
+        # self.static()
+        # self.index_static()
+        self.client.get("/")
+
+    @task(2)
+    def load_exercise(self):
+        # self.static()
+        # self.exercise_static()
+        response = self.client.get("/securesync/api/user/status")
+        user_id = json.loads(response.text)["user_id"]
+        # self.client.get("/static/khan-exercises/third_party/MathJax/2.1/MathJax.js?config=KAthJax-da9a7f53e588f3837b045a600e1dc439")
+        # self.client.get("/static/khan-exercises/exercises/addition_1.html")
+        # self.client.get("/math/arithmetic/addition-subtraction/basic_addition/e/addition_1")
+        self.client.get("/api/exercise/addition_1")
+        self.client.get("/api/exerciselog/?exercise_id=addition_1&user=" + user_id)
+        self.client.get("/api/attemptlog/?user=" + user_id + "&limit=10&exercise_id=addition_1&context_type__in=playlist&context_type__in=exercise")
+
+    @task(3)
+    def answer_exercise(self):
+        response = self.client.get("/securesync/api/user/status")
+        user_id = json.loads(response.text)["user_id"]
+        exerciselog_id = binascii.b2a_hex(os.urandom(15))
+        exerciselog_url = "/api/exerciselog/" + exerciselog_id + "/"
+        headers = {'Content-type': 'application/json', 'Accept': 'text/plain'}
+        url  = "/securesync/api/user/" + user_id + "/"
+        data = {"exercise_id":"addition_1","user":url ,"context_type":"exercise","context_id":"","language":"","version":"0.13.0","seed":82,"complete":True,"points":20,"response_count":1,"response_log":"[{\"type\":\"loaded\",\"timestamp\":\"2015-04-21T11:37:31.533\"},{\"type\":\"answer\",\"answer\":\"10\",\"correct\":True,\"timestamp\":\"2015-04-21T11:37:37.667\"}]","timestamp":"2015-04-21T11:37:31.533","correct":True,"answer_given":"10","time_taken":6134}
+        self.client.post("/api/attemptlog/", data=json.dumps(data), headers=headers)
+        data = {"attempts":5,"attempts_before_completion":5,"complete":True,"completion_counter":None,"completion_timestamp":"2015-04-10T13:35:39.940","counter":None,"deleted":False,"exercise_id":"addition_1","id":exerciselog_id,"language":None,"points":100,"resource_uri": exerciselog_url,"signature":None,"signed_version":1,"streak_progress":100,"struggling":False,"user": url}
+        self.client.put(exerciselog_url, data=json.dumps(data), headers=headers)
```

Comment:
You should check the response codes on this. I'm guessing this might 404, as it's a randomly generated id that won't yet exist in the DB.